core/executor/full_exit_nft_executor.go

```diff
@@ -128,45 +126,21 @@ func (e *FullExitNftExecutor) Prepare() error {
 	txInfo.CollectionId = exitNft.CollectionId
 
 	e.exitNft = exitNft
+	e.exitEmpty = isExitEmptyNft
 	return nil
 }
 
 func (e *FullExitNftExecutor) VerifyInputs(skipGasAmtChk bool) error {
-	bc := e.bc
```

Comment:
Is this because `FullExitNft` should always be executed? If that is the case, can you take some time to check other executor that does unnecessary verify, and fix it in another new PR.